doc/source/examples/unresolved-cfd-dem/gas-solid-spouted-bed/gas-solid-spouted-bed.rst

```diff
@@ -0,0 +1,404 @@
+            set density                        = 1
+          end
+    end
+
+
+Initial conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the initial conditions, we choose zero initial conditions for the velocity. 
+
+.. code-block:: text
+
+    subsection initial conditions
+        set type = nodal
+        subsection uvwp
+            set Function expression = 0; 0; 0; 0
+        end
+    end
+ 
+
+Boundary conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the boundary conditions, we choose a slip boundary condition on all the walls of the bed and the channel except the inlet at the bottom of the channel and the bottom of the bed and the outlet on the top of the bed where an outlet boundary conditions was imposed.  At the base of the channel and bottom walls of the bed, we impose a dirichlet boundary condition with an inlet velocity of 0.2 m/s and a background velocity of 1.25 respectively. For more information about the boundary conditions, please refere to the `Boundary Conditions Section <../../../parameters/cfd/boundary_conditions_cfd.html>`_
+
+.. code-block:: text
+
+   set time dependent         = false
+   set number                  = 4
+   	subsection bc 0
+     		set id		   = 0
+     		set type              = slip
+    	end
+   	subsection bc 1
+     		set id		   = 2
+     		set type              = outlet
+    	end
+   	subsection bc 2
+     		set id		   = 1
+     		set type              = function
+       		subsection u
+            		set Function expression = 0
+        	end
+        	subsection v
+            		set Function expression = 20
+        	end
+        	subsection w
+            		set Function expression = 0
+        	end
+       end
+       subsection bc 3
+     		set id		   = 3
+     		set type              = function
+        	subsection u
+            		set Function expression = 0
+       		end
+        	subsection v
+            		set Function expression = 1.25
+       	 	end
+        	subsection w
+            		set Function expression = 0
+        	end
+       end
+
+    end
+
+The additional sections for the CFD-DEM simulations are the void fraction subsection and the CFD-DEM subsection. These subsections are described in detail in the `CFD-DEM parameters <../../../parameters/unresolved-cfd-dem/unresolved-cfd-dem.html>`_ .
+
+Void fraction
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Since we are calculating the void fraction using the packed bed of the DEM simulation, we set the mode to "dem". For this, we need to read the dem files which we already wrote using check-pointing. We, therefore, set the read dem to "true" and specify the prefix of the dem files to be dem. We choose to use the quadrature centered method (QCM) to calculate the void fraction. This method does not require smoothing the void fraction as it is space and time continuous. For this simulation, we use a reference sphere having the same volume as the mesh elements as the averaging volume to calculate the void fraction.
```

Comment:
Just curious, why is the l2 smoothing factor is not 0 by default so we won't need to specify the 0 value for qcm?

Reply:
true we don't have to. I will remove it if it is already specified.